commons/clever-tools/ssh-access.md

```diff
@@ -0,0 +1,64 @@
+---
+title: SSH access
+shortdesc: SSH access to running instances
+tags:
+- cli-setup
```

Comment:
This should be in another category as well, seeing as it's not exclusive to clever-tools.